models/classes.ts

```diff
@@ -0,0 +1,14 @@
+import { object, array, number, string, TypeOf } from "yup";
+import { userSchema } from "./users";
+
+export const classSchema = object({
+  id: string().ensure().required(),
+  name: string().ensure().required(),
+  level: number().required(),
```

Comment:
So I read up on Yup a bit more, and found the following: 

`ensure()` will typecast nulls and undefined into empty strings
`defined()` means that all values except undefined are ok.
`required()` means all non null value. 

So if we want to define a value in our model that is a string and nullable, it would be like `id: string().nullable().defined()`. 

So, using that, we can remove the `ensure()'s` here. None of these values should be null for right now, so keep them as `required()`,
`